index.d.ts

```diff
@@ -75,6 +75,26 @@ declare const isStream: {
 	```
 	*/
 	transform(input: unknown): input is stream.Transform;
+
+	/**
+	@returns The [`type of stream`](https://nodejs.org/api/stream.html#stream_types_of_streams).
+
```

Comment:
Would be good to describe some use-cases for this method.

Reply:
done f4b590b